spec-tests/src/test/java/org/candlepin/spec/environments/EnvironmentSpecTest.java

```diff
@@ -466,7 +470,7 @@ public void shouldGiveConsumerContentFromMultipleEnvironments() {
         ConsumerDTO consumer = ownerClient.consumers().createConsumer(Consumers.random(owner)
             .environments(List.of(env1, env2, env3)));
         ApiClient consumerClient = ApiClients.ssl(consumer);
-        consumerClient.consumers().bindPoolSync(consumer.getUuid(), pool.getId(), 1);
+        consumerClient.consumers().bindPool(consumer.getUuid(), pool.getId(), 1);
```

Comment:
Every other invocation in the two spec files is to bindPoolSync. Why is this one different?

Reply:
Because it doesn't care about the response. `bindPoolSync` deserializes the synchronous version of the response. Since the test doesn't use the response I used `bindPool` to skip the deserialization.

Reply:
good enough